src/common/templates/_pdb.tpl

```diff
@@ -0,0 +1,47 @@
+{{/*
+Create Pod Distribution Budget Configurations
+Usage example:
+{{- include "harnesscommon.pdb.renderPodDistributionBudget" . }}
+*/}}
+{{- define "harnesscommon.pdb.renderPodDistributionBudget" -}}
+{{- $ := .ctx }}
+{{- if or $.Values.global.pdb.create $.Values.pdb.create }}
```

Comment:
what about a case where 90% of the services need pdb, and 10% dont.
should it have a higher precendence if local is set to true, it will always enable.

Reply:
That's one case where this will be more work. We cannot have local value set to true by default, because then PDB will always be enabled since local will have a precedence. For this scenario that will be the case, we set local to true default and the customer has to manually disable services they don't want. But then for the other way around when they need it for a few services, we have the same situation. So either way it looks like a trade-off, this would be more dependent on what the majority of customers will use, whether they have PDB enabled 
- for most services
- for some services
- disabled for all
- enabled for all